<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dmul</class>
 <widget class="QWidget" name="Dmul">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dmul</string>
  </property>
  <widget class="QGroupBox" name="gb_setup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>761</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Setup</string>
   </property>
   <widget class="QComboBox" name="c_dataType">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>eCtune</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AEM EMS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chrome 1.4+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hondata Kpro</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="c_portName">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>/tmp/ttyV0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>/tmp/ttyV1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CNCA0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CNCB0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>/dev/ttyUSB0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>/dev/ttyUSB1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM6</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="c_portSpeed">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>9600 baud</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200 baud</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400 baud</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200 baud</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lt_type">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lt_portName">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="lt_portSpeed">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port Speed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_update">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lt_title">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>5</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Datalog Emulator</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_values">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>761</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>Values</string>
   </property>
   <widget class="QSlider" name="vs_rpm">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>16</width>
      <height>241</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>12000</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="vs_ect">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>16</width>
      <height>241</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vs_iat">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>16</width>
      <height>241</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vs_tps">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>16</width>
      <height>241</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vs_map">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>16</width>
      <height>241</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vs_ign">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>16</width>
      <height>241</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vs_batt">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>49</y>
      <width>20</width>
      <height>241</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lt_rpm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>RPM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lt_ect">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>ECT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lt_iat">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>IAT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lt_tps">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>TPS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lt_map">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>MAP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lt_ign">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>IGN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lt_batt">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>BATT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_rpm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_ect">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_iat">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_tps">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_map">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_ign">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_batt">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>300</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="vs_vss">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>16</width>
      <height>241</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vs_inj">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>16</width>
      <height>241</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vs_O2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>16</width>
      <height>241</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vs_load">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>16</width>
      <height>241</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vs_baro">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>49</y>
      <width>20</width>
      <height>241</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lt_vss">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>VSS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lt_inj">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>INJ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lt_O2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>O2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lt_load">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>LOAD</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lt_baro">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>BARO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_vss">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_inj">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_O2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_load">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>300</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_baro">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>300</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>vs_rpm</sender>
   <signal>valueChanged(int)</signal>
   <receiver>l_rpm</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vs_ect</sender>
   <signal>valueChanged(int)</signal>
   <receiver>l_ect</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vs_iat</sender>
   <signal>valueChanged(int)</signal>
   <receiver>l_iat</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vs_tps</sender>
   <signal>valueChanged(int)</signal>
   <receiver>l_tps</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vs_map</sender>
   <signal>valueChanged(int)</signal>
   <receiver>l_map</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vs_ign</sender>
   <signal>valueChanged(int)</signal>
   <receiver>l_ign</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>617</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vs_batt</sender>
   <signal>valueChanged(int)</signal>
   <receiver>l_batt</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>729</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>727</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vs_vss</sender>
   <signal>valueChanged(int)</signal>
   <receiver>l_vss</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vs_inj</sender>
   <signal>valueChanged(int)</signal>
   <receiver>l_inj</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vs_O2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>l_O2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>607</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vs_load</sender>
   <signal>valueChanged(int)</signal>
   <receiver>l_load</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>667</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vs_baro</sender>
   <signal>valueChanged(int)</signal>
   <receiver>l_baro</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>729</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>727</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
